modules/dropdown/main/index.coffee

```diff
@@ -135,11 +140,29 @@ class Dropdown extends hx.EventEmitter
     if @isOpen() then @hide(cb) else @show(cb)
     this
 
+  dropdownContent: (dropdownContent, cb) ->
+    if arguments.length
+      setupDropdown = dropdownContentToSetupDropdown dropdownContent
+      @_.dropdownContent = dropdownContent
+      @_ = hx.shallowMerge @_, {
+        setupDropdown,
+        dropdownContent
+      }
+      @_.render cb
+    else
+      @_.dropdownContent
+
   show: (cb) ->
-    _ = @_
 
-    if not _.visible
-      _.visible = true
+    if not @_.visible
+      @_.visible = true
+      @render cb
+    this
+
+  render: (cb) ->
```

Comment:
most of the render method does what the show method previously did.

The render method does nothing if the dropdown is not visible (in particular, it will never call the callback)